### PR TITLE
Add spotless-maven-plugin configuration and use it to reformat the code

### PR DESCRIPTION
**Pull request by wborn**
This reformats the code using the code style that is also used with openHAB (see [static-code-analysis](https://github.com/openhab/static-code-analysis/tree/main/codestyle/src/main/resources)).

### CODE REVIEW

File: pom.xml
Comment (wborn) [inline pom.xml:1]: This commit exclude can be removed when https://github.com/jupnp/jupnp/pull/167 is merged.

File: pom.xml [hunk 1]
```diff
@@ -203,6 +209,53 @@
...
+          <version>${spotless.version}</version>
+          <configuration>
+            <java>
+              <eclipse>
+                <file>${basedirRoot}/tools/spotless/codestyle.xml</file>
+                <version>${spotless.eclipse.version}</version>
+              </eclipse>
+              <excludes>
+                <exclude>**/src/main/java/org/jupnp/util/io/Base64Coder.java</exclude>
+              </excludes>
```
Comment (wborn) [inline pom.xml:224]: This exclude can be removed when https://github.com/jupnp/jupnp/pull/167 is merged.

Comment (kaikreuzer) [reply pom.xml:224]: Then you can remove it now. :-)

Comment (wborn) [reply pom.xml]: Great! :+1:

**Review by kaikreuzer — APPROVED** [review]
Thanks!

**Pull request merged**